frontend/test/metabase-lib/lib/Question.unit.spec.js

```diff
@@ -1570,19 +1570,14 @@ describe("Question", () => {
   });
 
   describe("Question.generateQueryDescription", () => {
-    const mockTableMetadata = {
-      display_name: "Order",
-      fields: [{ id: 1, display_name: "Total" }],
-    };
-
     it("should work with multiple aggregations", () => {
       const question = base_question.setDatasetQuery({
         query: {
           "source-table": ORDERS.id,
-          aggregation: [["count"], ["sum", ["field", 1, null]]],
+          aggregation: [["count"], ["sum", ["field", 6, null]]],
         },
       });
-      expect(question.generateQueryDescription(mockTableMetadata)).toEqual(
+      expect(question.generateQueryDescription()).toEqual(
         "Orders, Count and Sum of Total",
       );
```

Comment:
`base_question` uses metadata from the [sample database fixture](https://github.com/metabase/metabase/blob/master/frontend/test/__support__/sample_database_fixture.json). [Field 1 there is actually "Created At"](https://github.com/metabase/metabase/blob/290b2db5e6dcfbff13c23481d3ed779aaa3d06e9/frontend/test/__support__/sample_database_fixture.json#L474), so changing the ID to 6 to correctly reference [the "Total" column](https://github.com/metabase/metabase/blob/290b2db5e6dcfbff13c23481d3ed779aaa3d06e9/frontend/test/__support__/sample_database_fixture.json#L591)